ROMFS/px4fmu_common/init.d-posix/airframes/1064_cupcar

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+#
+# @name Rover
+#
+
+. ${R}etc/init.d/rc.rover_defaults
+
+if [ $AUTOCNF = yes ]
+then
+	param set GND_L1_DIST 5
+	param set GND_SP_CTRL_MODE 1
+	param set GND_SPEED_D 0.001
+	param set GND_SPEED_I 3
+	param set GND_SPEED_IMAX 0.125
+	param set GND_SPEED_P 0.25
+	param set GND_SPEED_THR_SC 1
+	param set GND_SPEED_TRIM 4
+	param set GND_THR_CRUISE 0.3
+	param set GND_THR_IDLE 0
+	param set GND_THR_MAX 0.5
+	param set GND_THR_MIN 0
+
+	param set MIS_LTRMIN_ALT 0.01
+	param set MIS_TAKEOFF_ALT 0.01
+	param set NAV_ACC_RAD 0.5
+	param set NAV_LOITER_RAD 2
+
+	param set CBRK_AIRSPD_CHK 162128
+
+	param set GND_MAX_ANG 0.6
+	param set GND_WHEEL_BASE .15
+
+fi
+
+set MAV_TYPE 10
+
+set MIXER_FILE etc/mixers-sitl/rover_ackermann_sitl.main.mix
```

Comment:
Could you separate the addition of a new SITL model to a separate PR? The changes seem quite orthogonal

Reply:
I wanted to simplify and tie together the changes also in PX4-SITL_gazebo for enabling the cupcar by including it in this PR as it was added to the generator list.